### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory: ''

### ISSUE THREAD

**Issue by ws-choi** [zkx06111/WSRGlow#8]
Hi, authors,
Thank you for open sourcing this great repository.

I ran ```python train.py --config config.yaml```, and got this error: FileNotFoundError: [Errno 2] No such file or directory: ''

```
Traceback (most recent call last):
  File "/home/wschoi/PycharmProjects/WSRGlow/train.py", line 345, in <module>
    WaveGlowTask4.start()
  File "/home/wschoi/PycharmProjects/WSRGlow/train.py", line 274, in start
    period=1 if hparams['save_ckpt'] else 100000
  File "/home/wschoi/PycharmProjects/WSRGlow/training_utils.py", line 23, in __init__
    os.makedirs(filepath, exist_ok=True)
  File "/home/wschoi/miniconda3/envs/wsrglow/lib/python3.7/os.py", line 223, in makedirs
    mkdir(name, mode)
FileNotFoundError: [Errno 2] No such file or directory: ''

Process finished with exit code 1
```

I guess this error was occurred because ```args_work_dir``` was set to '' unless ```args.exp_name``` is not a default value.

https://github.com/zkx06111/WSRGlow/blob/1b8fc4939c72b319efdb520ba2868eacf468ca18/hparams.py#L39-L42
 
and then, ``` hparams_['work_dir']``` is set to ```args_work_dir``` regardless of ```work_dir``` of ```config.yaml```.

https://github.com/zkx06111/WSRGlow/blob/1b8fc4939c72b319efdb520ba2868eacf468ca18/hparams.py#L84-L86

---

***TLDR***;
 
This error is occurred only when ```args.exp_name == '' ```.

For those who want to quickly reproduce ```train.py``` I would recommend a script like below.

```python train.py --config config.yaml --config config.yaml --exp_name WSRGlow```
